src/test/SmartComponents/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -142,6 +142,24 @@ describe('Step Registration', () => {
         const [ , , cancel ] = verifyButtons();
         verifyCancelButton(cancel);
     });
+
+    test('should allow choosing activation keys', () => {
+        screen.
+        getByLabelText('Embed an activation key and register systems on first boot').
+        click();
+
+        const organizationId = screen.getByLabelText('Organization ID');
+        expect(organizationId).not.toHaveValue();
+        expect(organizationId).toBeDisabled();
+
+        // can't getByLabelText b/c the label contains an extra <span>
+        // with a `*` to denote required
+        const activationKey = screen.getByTestId('subscription-activation');
+        expect(activationKey).toHaveValue('');
+        expect(activationKey).toBeEnabled();
+        // todo: should this control be required ???
+        // expect(activationKey).toBeRequired();
```

Comment:
@Gundersanne there's no `required` attribute on the input element and I think this is what the test framework looks for. Should there be one ?

Reply:
yea I think so.